src/libstore/globals.hh

```diff
@@ -672,7 +682,7 @@ public:
           send a series of commands to modify various settings to stdout. The
           currently recognized commands are:
 
-            - `extra-sandbox-paths`  
+            - `extra-sandbox-paths`
```

Comment:
IIRC this trailing whitespace is significant, it causes it to be rendered as a definition list.

Reply:
Aaaaaargh thanks. Fixed in a21de5181a7a5e6377e63c749a1d7f0758c57eb8